test/lib/absinthe/relay/connection_test.exs

```diff
@@ -188,7 +189,7 @@ defmodule Absinthe.Relay.ConnectionTest do
       end
 
       @desc "The favorite pets for a person"
-      connection field :favorite_pets, connection: :favorite_pets do
+      connection field :favorite_pets, node_type: :pet, connection: :favorite_pets do
```

Comment:
I'm also not 100% sure, but I think node_type always had to be defined, and I'm quite sure that the node type here was never valid but the `connection field` didn't check this(?) I remember this was causing issues while I was writing the test, but can't remember why.

Is this a valid syntax (without specifying `node_type`)?